src/main/java/Database/users.csv

```diff
@@ -0,0 +1,2 @@
+User1,Password1,Playlist 1;Playlist 2
```

Comment:
Lol. See other comment in regards to that semicolon separator

Reply:
Same thing :(. The way I have my parser written right now is that it takes a line at a time, splits by commas, and then I further split the subdata if necessary (using their respective delimiter).